manim/mobject/three_d/three_dimensions.py

```diff
@@ -555,6 +555,8 @@ def generate_points(self) -> None:
         for dim, value in enumerate(self.dimensions):
             self.rescale_to_fit(value, dim, stretch=True)
 
+    init_points = generate_points
+
```

Comment:
I'd prefer if instead of re-assigning the `init_points` method for `Prism`, we'd fix the improper implementation of this in the `Cube` class to read

```py
    def init_points(self) -> None:
        self.generate_points()
```
instead of the same sort of assignment here. This fixes the problem properly for all subclasses.

Reply:
Thank you. Fixed!